src/tests/unit/SlackRTMHandlerTest.ts

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// tslint:disable: no-unused-expression
```

Comment:
please don't disable the linter - it's serving a purpose, and this rule is not a good reason to disable it.

Reply:
Fair in this case, but I usually disable this rule to get around tslint complaining that something like `expect(foobar).to.be.defined;` isn't correct, despite it being a `get` function and having side effects.